src/Pages/juyoungson/Main/FeedBox/FeedFooterBox/Comment/Comment.js

```diff
@@ -0,0 +1,40 @@
+import React, { Component } from "react";
+import {Link} from 'react-router-dom';
+import "./Comment.scss";
+
+class Comment extends Component {
+  deleteCommentBtn = () => {
+    this.props.deleteCommentBtn();
+  };
+
+  chgLikeColor = () => {
+    this.props.chgLikeColor();
+  };
+
+  render() {
+    const { commentList, deleteCommentBtn, chgLikeColor } = this.props;
+    return (
+      <div className="commentWrapper">
+        {commentList.map((comment) => {
+          return (
+            <div className="commentBox">
+              <div key={comment.id}>
+                <Link to="/mains">{`${comment.uesrName}`}</Link>
+                <span>{`${comment.value}`}</span>
+              </div>

```

Comment:
```suggestion
              <div key={comment.id}>
                <Link to="/mains">{comment.uesrName}</Link>
                <span>{comment.value}</span>
              </div>

```
바로 넣어주셔도 되는데 저렇게 하신 이유가 있나요??

Reply:
바로 넣으려고 했는데 안되가지고 이것 저것 시도해보다가 문법이 잘못 된것이 아니라 userName 인데 uesrName으로 해놔서 안나왔던 거였습니당.